packages/vaults/src/transaction/member/member_remove.ts

```diff
@@ -1,4 +1,4 @@
-import {TransactionType, VaultRole} from "../../enum/enums";
+import {TransactionType} from "../../enum/enums";
```

Comment:
I think it should format it to `{ TransactionType }` automatically

Reply:
yep, fixed